### PR TITLE
Rdcc 641 - Pact

### PR DESCRIPTION
**Pull request by ConorH994**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-641

### Change description ###

PACT

### CODE REVIEW

File: src/contractTest/java/uk/gov/hmcts/reform/userprofileapi/IdamConsumerTest.java [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
...DT3ROb290ZjF4TWg0bGc3MFlBPSIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmcmVnLXRlc3QtdXNlci1ZOHlqVURSeWpyQGZlZW1haWwuY29tIiwiYXV0aF9sZXZlbCI6MCwiYXVkaXRUcmFja2luZ0lkIjoiYTU2MDliYjYtYzEzYi00MjQ0LTg3ODItNDNmZGViMDZlMDBjIiwiaXNzIjoiaHR0cHM6Ly9mb3JnZXJvY2stYW0uc2VydmljZS5jb3JlLWNvbXB1dGUtaWRhbS1hYXQuaW50ZXJuYWw6ODQ0My9vcGVuYW0vb2F1dGgyL2htY3RzIiwidG9rZW5OYW1lIjoiYWNjZXNzX3Rva2VuIiwidG9rZW5fdHlwZSI6IkJlYXJlciIsImF1dGhHcmFudElkIjoiYWNjNmUyYTAtMWExYi00OGM3LWJmZGItNzI1NjllM2E1NjkzIiwiYXVkIjoicmQtcHJvZmVzc2lvbmFsLWFwaSIsIm5iZiI6MTU2OTQ0MTkxMSwiZ3JhbnRfdHlwZSI6ImF1dGhvcml6YXRpb25fY29kZSIsInNjb3BlIjpbIm9wZW5pZCIsInByb2ZpbGUiLCJyb2xlcyIsImNyZWF0ZS11c2VyIiwibWFuYWdlLXVzZXIiXSwiYXV0aF90aW1lIjoxNTY5NDQxOTExMDAwLCJyZWFsbSI6Ii9obWN0cyIsImV4cCI6MTU2OTQ1NjMxMSwiaWF0IjoxNTY5NDQxOTExLCJleHBpcmVzX2luIjoxNDQwMCwianRpIjoiY2Q5MWM0NjQtMzU0Zi00N2I2LTkwYTUtNWY2Y2U3NGUwYTY5In0.aLobAYYCxkmryzKV1stmag63h-ndxrDjO4462YERcLDIXVmvFJNXfdPRg9U8WGv0GkOrSkHVJ7tbdLQySnOVYulXkPl71g5MqU7ZuEQvHaBpfW9exBCfP-pw8kWyMUck-rB00tkEX7ZpS6euQM0WVbdczPnClxR3tWwktPfN-bCo6PPwqiMkC1DgTmjQBMtjgP1nEiJM7Kocqb2X3OCItf4lps1_nSG68jI98fwaLn8WQgk1sw9eebskChXDfpmIyreeGFWpHNpdFqOFfYEC5FnSgXHQw7Eu-hc5RofPZzKFrbwZHC31t5guK9Wq8zn9Xwe6743g4ozm3EHN8fsjVQ"; //todo
```
Comment (ConorH994) [inline src/contractTest/java/uk/gov/hmcts/reform/userprofileapi/IdamConsumerTest.java]: dummy text

File: build.gradle [hunk 2]
```diff
@@ -34,11 +35,12 @@ def versions = [
     reformHealthStarter: '0.0.4',
     reformLogging      : '5.1.0',
     reformS2sClient    : '3.0.0',
-    serenity           : '2.0.11',
+    serenity           : '2.0.23',
     sonarPitest        : '0.5',
     springBoot         : '2.1.4.RELEASE',
     springHystrix      : '2.1.1.RELEASE',
```
Comment (shashank-rastogi) [inline build.gradle]: is there a requirement to upgrde Serenity and Dependency check  for this PR?

File: build.gradle [hunk 3]
```diff
@@ -315,13 +343,20 @@ dependencies {
...
+    testPactCompile sourceSets.main.runtimeClasspath
+    testPactCompile sourceSets.test.runtimeClasspath
 }
 
 // https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html
 dependencyCheck {
     // Specifies if the build should be failed if a CVSS score above a specified level is identified.
     // range of 0-10 fails the build, anything greater and it doesn't fail the build
     failBuildOnCVSS = System.getProperty('dependencyCheck.failBuild') == 'true' ? 0 : 11
+    println 'Hi from dependencyCheck'
```
Comment (shashank-rastogi) [inline build.gradle]: can you remove the println statements?

File: build.gradle (hunk 2 above)
Comment (ConorH994) [reply build.gradle]: It was updated in master using one of the dependabot PR's while I was working on this branch. I'm not sure how dependabot works but the change was made on my PR too.

File: build.gradle (hunk 3 above)
Comment (ConorH994) [reply build.gradle]: Yea

**Pull request merged**